src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -563,9 +580,33 @@ void RenderTarget::applyCurrentView()
 {
     // Set the viewport
     const IntRect viewport = getViewport(m_view);
-    const int     top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
+    int           top      = static_cast<int>(getSize().y) - (viewport.top + viewport.height);
     glCheck(glViewport(viewport.left, top, viewport.width, viewport.height));
 
+    // Set the scissor rectangle and enable/disable scissor testing
+    const FloatRect scissor = m_view.getScissor();
+
+    if ((scissor.left == 0) && (scissor.top == 0) && (scissor.width == 1) && (scissor.height == 1))
+    {
+        if (m_cache.scissorEnabled)
+        {
+            glCheck(glDisable(GL_SCISSOR_TEST));
+            m_cache.scissorEnabled = false;
+        }
```

Comment:
I feel like this "resetting"/disabling should be documented somehow on the API

Reply:
Documented in `setScissor` documentation of latest commit.